pkgs/by-name/ic/icestudio/package.nix

```
@@ -5,20 +5,21 @@
   buildNpmPackage,
   makeDesktopItem,
   makeWrapper,
+  unstableGitUpdater,
 
   nwjs,
   python3,
 }:
 
 let
   # Use unstable because it has improvements for finding python
-  version = "0-unstable-2024-11-18";
+  version = "0-unstable-2025-03-08";
```

We need to use the [latest release tag](https://github.com/FPGAwars/icestudio/tags), else the update script will fail. The commit and PR titles will need to be updated as well.

```suggestion
  version = "0.12-unstable-2025-03-08";
```

---

Okay..i will fix that now